python3/examples/setup.cfg

```
@@ -20,9 +20,3 @@ classifiers =
 [options]
 packages = find:
 python_requires = >=3.6.*, <4
```

Pip doesn't like this line, and will refuse to build the package, instead use:
```suggestion
python_requires = >=3.6, <4
```